packages/x-data-grid-pro/src/tests/detailPanel.DataGridPro.test.tsx

```diff
@@ -86,10 +86,10 @@ describe('<DataGridPro /> - Detail panel', () => {
     await microtasks();
 
     const virtualScrollerContent = $('.MuiDataGrid-virtualScrollerContent')!;
-    expect(virtualScrollerContent).toHaveInlineStyle({
-      width: 'auto',
+    expect(virtualScrollerContent).toHaveComputedStyle({
```

Comment:
I changed `height` to `flex-basis` in https://github.com/mui/mui-x/pull/14966/files#diff-74adfff8c7dc341a80f5a489eb850b9d566b1be041dfdf0e47a9848609f5ac53R537, and I makes more sense to check the actual height rather than specific inline style.

Reply:
Comment not specific to this in particular, but I feel like we often test the implementation too closely, e.g. we set a specific style and test for that style. Quite a few of our tests would be better as visual regression tests, they would be more resilient to implementation changes, and would catch actual regressions (like this one could have been caught if we had the visual regression test).